ui/epicSkills/epicSkills.scss

```diff
@@ -15,6 +15,7 @@
     40px // Base
   );
   @include epic-table-grid('skill', $skill-grid-template);
+  @include epic-table-grid-row-border('skill');
```

Comment:
Ideally, we'd apply row borders for all table grids, but this doesn't work for tables with hidden rows, since they hide rows causing multiple borders between rows like:
![image](https://user-images.githubusercontent.com/5629800/212576067-15cee799-0b42-4e23-9654-efb33509c53d.png)